### DISCUSSION TITLE
How can I create space using key tab?

### DISCUSSION THREAD

**Discussion by JenuelDev** [ueberdosis/tiptap#2008] [General Questions]
We are using tiptap on our project, but I have trouble recreating the tab effect. So that if I press the tab, it should create space, and doesn't leave in the editor.

**Comment by cadars** [comment]
By doing so you would break keyboard tabbing, the most basic accessibility requirement, but maybe you could adapt the SmilieReplacer extension from https://tiptap.dev/examples/savvy ? (not sure that would work with tab)

**Reply by JenuelDev** [reply]
thanks for answering my question, I appreciate your time. Sure I'll try.

**Comment by philippkuehn** [comment]
you can use `addKeyboardShortcuts`: https://tiptap.dev/guide/custom-extensions/#keyboard-shortcuts

**Reply by JenuelDev** [reply]
thanks for answering my question, I appreciate your time. Sure I'll try.
